x/auth/genaccounts/genesis_state.go

```diff
@@ -10,23 +10,15 @@ import (
 )
 
 // State to Unmarshal
-type GenesisState struct {
-	Accounts GenesisAccounts `json:"accounts"`
-}
-
-// NewGenesisState creates a new GenesisState object
-func NewGenesisState(accounts GenesisAccounts) GenesisState {
-	return GenesisState{
-		Accounts: accounts,
-	}
-}
+type GenesisState GenesisAccounts
```

Comment:
this looks a bit odd

Reply:
Although it is a bit weird in a narrow context, I think it's probably fine as it stays consistent with the other modules and the associated pattern. But yeah it _could_ be changed. I think I prefer it as it is, but no very strong sense either way.

Reply:
This is just a type alias. Every module has a `GenesisState` type. However, in the `x/auth/genaccounts` sub-module, it contains no really data apart from accounts, so there is no need to create an actual type that nests these accounts. So we can ditch `GenesisState` all together and just use `GenesisAccounts` or leave as-is.